<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Fondo</class>
 <widget class="QWidget" name="Fondo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1240</width>
    <height>582</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
#Fondo{
	background-color: #fff;
}
#fondoTitulo{
	background-color:#03A9F4;
	box-shadow: 10px 5px 5px black;
}
#fondoTitulo_2{
	background-color:#03A9F4;
	box-shadow: 10px 5px 5px black;
}

#titulo{
	font-size: 25px;
    color:#fff;
	font-family: Arial, Verdana;
	font-weight: bold;
}

#btnGuardar{ background: #2E7D32; color: #fff; font-size: 12px; font-weight: bold; }
#btnEliminar{ background: #B71C1C; color: #fff; font-size: 12px; font-weight: bold; }
#btnLimpiar{ background: #0277BD; color: #fff; font-size: 12px;font-weight: bold;  }
#btnBuscarImg{ background: #0277BD; color: #fff; font-size: 12px; font-weight: bold; }

#btnBuscar{ background: #2E7D32; color: #fff; font-size: 12px; font-weight: bold; }

#btn_menu1{ background: #0277BD; color: #fff; font-size: 12px;font-weight: bold;  }
#btn_menu2{ background: #0277BD; color: #fff; font-size: 12px;font-weight: bold;  }
#btn_menu3{ background: #0277BD; color: #fff; font-size: 12px;font-weight: bold;  }
#btn_menu4{ background: #0277BD; color: #fff; font-size: 12px;font-weight: bold;  }
#btn_menu5{ background: #0277BD; color: #fff; font-size: 12px;font-weight: bold;  }

#verImagen{
	background-color:#FAFAFA;
}</string>
  </property>
  <property name="inputMethodHints">
   <set>Qt::ImhNone</set>
  </property>
  <widget class="QFrame" name="fondoTitulo">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1241</width>
     <height>91</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="titulo">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>30</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Registro de Pacientes</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>91</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>img/logo.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QTableWidget" name="listaEmpleados">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>200</y>
     <width>1091</width>
     <height>361</height>
    </rect>
   </property>
   <property name="cursor" stdset="0">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="statusTip">
    <string/>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="dragDropOverwriteMode">
    <bool>true</bool>
   </property>
   <property name="rowCount">
    <number>0</number>
   </property>
   <property name="columnCount">
    <number>23</number>
   </property>
   <column>
    <property name="text">
     <string>ID</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Cédula</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Nombres</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Apelidos</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Fecha de Nacimiento</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Edad</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string># Aportaciones</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Dirección 1</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Dirección 2</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Teléfono 1</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Teléfono 2</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Email</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>New Column</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Días Laborales</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Género</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Nivel Académico</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string># Cuenta Bancaria</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Tipo de Discapacidad</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Nombre Recomendado</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Teléfono Recomendado</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Celular Recomendado</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Ciudad</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Foto</string>
    </property>
   </column>
  </widget>
  <widget class="QFrame" name="fondoTitulo_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>90</y>
     <width>111</width>
     <height>581</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#fondoTitulo2{
	background-color:#03A9F4;
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="btn_menu1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>PACIENTE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_menu2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>RESERVAS</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_menu3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>MENÚ</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_menu4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>MENÚ</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_menu5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>MENÚ</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btnEliminar">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>120</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Eliminar</string>
   </property>
  </widget>
  <widget class="QWidget" name="formLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>120</y>
     <width>153</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="frmUsuarios_4">
    <property name="leftMargin">
     <number>12</number>
    </property>
    <property name="topMargin">
     <number>12</number>
    </property>
    <property name="rightMargin">
     <number>12</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item row="0" column="1">
     <widget class="QComboBox" name="cbxBusq">
      <item>
       <property name="text">
        <string>Cédula</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Apellidos</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Nombres</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Ciudad</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="lblBusq_2">
      <property name="text">
       <string>Buscar Por:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="formLayoutWidget_5">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>120</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="frmUsuarios_6">
    <property name="leftMargin">
     <number>12</number>
    </property>
    <property name="topMargin">
     <number>12</number>
    </property>
    <property name="rightMargin">
     <number>12</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QPushButton" name="btnBuscar">
      <property name="font">
       <font>
        <pointsize>-1</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Buscar</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="txtBusq"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="btnLimpiar">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>120</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Nuevo</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
